clients/vscode/src/chat/WebviewHelper.ts

```diff
@@ -350,7 +350,12 @@ export class WebviewHelper {
     try {
       await this.client?.updateActiveSelection(context);
     } catch {
-      this.logger.warn("Active selection sync failed. Please update your Tabby server to the latest version.");
+      this.logger.logEveryN(
+        "sync-active-selection-failed",
+        100,
+        "warn",
+        "Active selection sync failed. Please update your Tabby server to the latest version.",
+      );
```

Comment:
```suggestion
      this.logger.logEveryN({
        key: "sync-active-selection-failed",
        log_every_n: 100,
        level: "warn",
        message: "Active selection sync failed. Please update your Tabby server to the latest version.",
      });
```

Provide reasonble default in function, so user can input only key, every, and message if needed